
[Executed at: Wed Mar 14 1:44:49 PDT 2018]

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/submission.py", line 20, in <module>
    imported=getattr(__import__(work_package, fromlist=fun_name), f)
AttributeError: module 'q5_work' has no attribute 'nroot_complex'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24781_3/asnlib/submission.py", line 22, in <module>
    print('Error: importing '+fun_name+'.')
TypeError: Can't convert 'list' object to str implicitly
Command exited with non-zero status 1
